packages/utils/babel-preset-env/src/index.js

```diff
@@ -1,6 +1,6 @@
 const semver = require('semver');
 
-const COMPATIBLE_PARCEL_BABEL_TRANSFORMER_SEMVER = '^2.0.0-alpha.1.1';
+const COMPATIBLE_PARCEL_BABEL_TRANSFORMER_SEMVER = '2.0.0-beta.1';
```

Comment:
Wouldn't this fail with the nightly?
```
semver.satisfies("2.0.0-nightly.1234", "2.0.0-beta.1") === false
```

Reply:
Ah yeah, we'll either need something permissive or rewrite this whenever lerna versions things